index.html

```diff
@@ -2,25 +2,56 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="login.css" />
-    <script src="login.js"></script>
-    <title>SIGN UP/SIGN IN FORM </title>
+    <title>LOG IN</title>
+
+    <!-- FONT -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Use the same font to the dashboard as well.

Reply:
Made the changes.